ui/packages/catalog-ui-search/src/main/webapp/react-component/container/metacard-interactions/dropdown.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define*/
+import * as React from 'react'
+import MarionetteRegionContainer from '../marionette-region-container'
+import MetacardInteractions from '.'
+import { hot } from 'react-hot-loader'
+import styled from '../../styles/styled-components'
+const Marionette = require('marionette')
+const CustomElements = require('../../../js/CustomElements.js')
+require('../../../behaviors/dropdown.behavior')
+
+const MetacardInteractionsDropdown = Marionette.ItemView.extend({
+  template() {
+    const props = {
+      model: this.model,
+      onClose: () => {
+        this.$el.trigger(`closeDropdown.${CustomElements.getNamespace()}`)
+      },
+    }
+    return <MetacardInteractions {...props} />
+  },
+  tagName: CustomElements.register('metacard-interactions-dropdown'),
+  handleShare() {},
+})
+
+// TODO: Add onHover from the + button
+const Button = styled.button`
+  display: 'inline-block';
+  text-align: center;
+  vertical-align: top;
+  width: ${props => props.theme.minimumButtonSize};
+  height: ${props => props.theme.minimumButtonSize};
+  line-height: ${props => props.theme.minimumButtonSize};
+`
+
+const ContainerView = Marionette.ItemView.extend({
```

Comment:
Any chance you can use https://github.com/codice/ddf/blob/master/ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/dropdown/dropdown.tsx ? Not sure if this file can be made kinda like this: https://github.com/codice/ddf/blob/master/ui/packages/catalog-ui-search/src/main/webapp/react-component/container/map-settings/map-settings.tsx